server/service/src/purchase_order_line/insert/generate.rs

```diff
@@ -21,6 +21,12 @@ pub fn generate(
         purchase_order_id: input.purchase_order_id,
         line_number,
         item_link_id: input.item_id,
+        requested_number_of_units: input
+            .requested_number_of_units
+            .unwrap_or(Default::default()),
+        requested_pack_size: input.requested_pack_size.unwrap_or(Default::default()),
```

Comment:
Can we default to the item's default packsize?

Rather than `.unwrap_or(Default::default()),` we should use `.unwrap_or_default()`